README.md

```diff
@@ -132,13 +132,13 @@ return a new object, leaving the original object unaffected:
 $ten = BigInteger::of(10);
 
 echo $ten->plus(5); // 15
-echo $ten->multipliedBy(3); // 30
+echo $ten->multipliedBy(3); // 45
```

Comment:
Curse you, immutability! My bad, I was intending to correct the other one, glimpsed this one and mistakenly thought it was supposed to demonstrate equivalence and "corrected" this one as well.